shanghai/main.py

```diff
@@ -14,48 +14,65 @@
 
 def exception_handler(loop, context):  # pylint: disable=unused-argument
     f = io.StringIO()
-    print("exception_handler context:", file=f)
+    print("=== Unhandled Exception ===", file=f)
+    print("- Context -", file=f)
     pprint(context, stream=f)
-    traceback.print_exc(file=f)
```

Comment:
I'll add an `isinstance` check and add log output for when this is not the case.